index.html

```diff
@@ -49,5 +49,169 @@ <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
+        <h2>Tonic</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+    </section>
+    
+    <div class="bg-white">
+      <section id="aboutme">
+        <div class="me">
+          <h3 class="aboutme">About Myself</h3>
+          <p class="p">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+  
+          <h4 class="h4-connect">LET'S CONNECT</h4>
+  
+          <ul class="flex item-center">
+            <li class="icon-tag">
+              <a href="#"><img src="./image/twitter.png" alt="twitter" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/Linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/m.png" alt="m" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/git.png" alt="Github" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/angelist.png" alt="angelist" /></a>
+            </li>
```

Comment:
- [ ] [OPTIONAL] I think you would like visitors who would like to contact you for opportunities to reach you. If that is you, you can give users more thrilling experience by providing your profile links or username in the `href` attribute of this social media icons so that can like, follow, connect or consult you.

Reply:
Thanks for the review @AbayomiOlaoye